src/main/java/umk/mat/pajda/ProjektZespolowy/services/ReviewService.java

```diff
@@ -107,4 +109,25 @@ public Boolean patchSelectReview(ReviewPatchPostDTO reviewPatchPostDTO, int id)
     }
     return true;
   }
+
+  public boolean validateTime(ReviewPatchPostDTO reviewPatchPostDTO) {
+    Review review = null;
+    LocalDateTime currentDateTime = LocalDateTime.now();
+    try {
+      review =
+          reviewRepository.findFirstByHashRevIDOrderByCreatedAtDesc(
```

Comment:
If we give review to the first waiter, the timer shouldn't affect giving review to other waiters